lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -377,12 +528,14 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
 
         if (defined(this.layers)) {
             var layers = this.layers.split(',');
+            var styles = (this.styles || this.parameters.styles || '').split(',');
```

Comment:
If it makes sense for the person constructing the json to specify `layers` and `styles` in the same way, yep. Or are they different enough concepts that there wouldn't be any great risk of catalog-constructor-error with `styles` as an array?
I'm guessing that you felt there was enough reason to do it like layers in the first place, so I'll go with that :-)